src/pcapi/core/mails/transactional/users/accepted_as_beneficiary_email.py

```diff
@@ -0,0 +1,60 @@
+from typing import Union
+
+from pcapi.core import mails
+from pcapi.core.mails.transactional.sendinblue_template_ids import SendinblueTransactionalEmailData
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.core.users.models import User
+from pcapi.core.users.models import UserRole
+from pcapi.models.feature import FeatureToggle
+
+
+def get_accepted_as_beneficiary_email_data(user: User) -> Union[dict, SendinblueTransactionalEmailData]:
+    if user.active_deposit is None:
+        raise ValueError("Beneficiary should have a deposit")
+    deposit_amount = user.active_deposit.amount
+
+    if not FeatureToggle.ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS.is_active():
+        return {
+            "Mj-TemplateID": 2016025,
+            "Mj-TemplateLanguage": True,
+            "Mj-campaign": "confirmation-credit",
+            "Vars": {
+                "depositAmount": int(deposit_amount),
+            },
+        }
+
+    return SendinblueTransactionalEmailData(
+        template=TransactionalEmail.ACCEPTED_AS_BENEFICIARY.value,
+        params={
+            "CREDIT": int(deposit_amount),
+        },
+    )
+
+
+def get_accepted_as_underage_beneficiary_email_data(user: User) -> SendinblueTransactionalEmailData:
+    if user.active_deposit is None:
+        raise ValueError("Beneficiary should have a deposit")
+    deposit_amount = user.active_deposit.amount
+
+    if FeatureToggle.ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS.is_active():
```

Comment:
ça ne devrait pas être 
```suggestion
    if not FeatureToggle.ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS.is_active():
```
?